lib/vscode_helper.dart

```diff
@@ -29,7 +29,7 @@ void updateLaunchJsonFile(FlutterProject project, Uri observatoryUri) {
   if (project.directory.basename == 'example') {
     final FlutterProject parentProject =
         FlutterProject.fromDirectory(project.directory.parent);
-    if (parentProject.isPlugin || parentProject.isModule) {
+    if (parentProject.pubspecFile.existsSync() && parentProject.hasExampleApp) {
```

Comment:
Isn't `parentProject.hasExampleApp` always true is this scope?

Reply:
Oh, technically, yes.